runtime/lua/vim/lsp/buf.lua

```diff
@@ -211,7 +211,7 @@ end
 --- Table must contain `start` and `end` keys with {row,col} tuples using
 --- (1,0) indexing.
 --- Can also be a list of tables that contain `start` and `end` keys as described above,
---- in which case `textDocument/rangesFormatting` support is required.
+--- in which case `textDocument/rangesFormatting` support is required.lspbuf
```

Comment:
is this an artifact?

Reply:
Also, please update the commit message / title accordingly as we no longer have an boolean option.